golos.publication/golos.publication.cpp

```diff
@@ -257,6 +270,7 @@ void publication::close_message(account_name account, uint64_t id) {
 
     tables::reward_pools pools(_self, _self);
     auto pool = get_pool(pools, mssg_itr->date);
+    fix_balances(pools, pool->state.funds.symbol);
```

Comment:
I consider quietly correcting balances in the pool of rewards - a completely wrong decision. Thus, we hide the problem, but do not signal about it. I agree that testing can be done at each closing post. But in the event of a mismatch, it is much better to signal a problem, for example, not to close posts. In this case, it will be seen that something went wrong and need to take certain actions about fix problem (look for where the money went, or a problem in the code).

Reply:
U r right, we have to save some sort of labels in cases of such inconsistency. However, I don't think that this information should be placed in a table with messages, because it is not directly related to them. It seems like for cases like this (something unexpected happened, but interruption of the action is not required) we can use a special table with warnings, what do you think about it?